### DISCUSSION TITLE
Detour: Filtering offmesh connections

### DISCUSSION THREAD

**Discussion by sweet235** [recastnavigation/recastnavigation#731] [Q&A]
Hopefully this is not a stupid question. I am using Detour's `dtQueryFilter`. It is straightforward to use it to ignore some navmesh tiles. Now I am trying to make it ignore some offmesh connections, but so far without success.

The game I am working on relies on offmesh connections for certain "special" moves a bot can do. Bots can use a jetpack to fly to higher places. But they can only do this as long they have enough fuel left. Now I am trying to make the path search ignore the corresponding offmesh connections once there is no fuel left. Is this possible?

**Comment by ZA139** [comment]
have u tried tag these offmesh linked tiles with different custom flags?
Using custom dtQueryFilter to ignore these tiles when there isn't enough fuel.
